.github/workflows/deployment.yml

```diff
@@ -12,23 +12,26 @@ on:
   push:
     branches:
       - main
+    tags:
+      - 'region-*'
```

Comment:
@PaulGoldschmidt Wouldn't a workflow dispatch make more sense here?

https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch

For now you could just deploy on any commit on this branch to test, later in a workflow dispatch would be the best thing to at least test a deployment to one location? In the end we should aim to mirror the ENGAGE-HF setup.